sycl/include/sycl/ext/intel/fpga_device_selector.hpp

```diff
@@ -22,6 +22,29 @@ class platform;
 namespace ext {
 namespace intel {
 
+namespace detail {
+// Scores a device by platform name. Uses const char * for the platform name to
+// avoid repeated string copying.
+inline int SelectDeviceByPlatform(const char *required_platform_name,
```

Comment:
We already use mix of 2 coding standards in SYCL headers depending on situation:
1) clang standard:     int Variable; doSomething();
2) std c++:                int variable; do_something();
This line doesn't follow any of those.
```suggestion
inline int selectDeviceByPlatform(const char *required_platform_name,
```

Reply:
Good point! I have changed the names of `SelectDeviceByPlatform` and `EnableFPGASimulator` to follow the LLVM coding standard.